docker/monitoring/prometheus-micrometer.yaml

```diff
@@ -0,0 +1,6 @@
+scrape_configs:
+  - job_name: 'Metrics Scraper'
```

Comment:
```suggestion
  - job_name: 'datahub-gms'
```

space is allowed in job names, but usually they are in snake case or kebab case.